primitives/avail/src/header/mod.rs

```diff
@@ -348,6 +348,23 @@ mod tests {
 		}
 	}
 
+	/// The `commitment.data_root is none`.
+	fn header_v2() -> Header<u32, BlakeTwo256> {
+		let commitment = v2::KateCommitment {
```

Comment:
Can we add a negative test here? Unless you feel that should be part of a diff PR for the other bugfix.

Reply:
Added  a corrupted header test at line 432